### ISSUE TITLE
Add friendly display name for the Blazor WebAssembly debugging extension for VSCode

### ISSUE THREAD

**Issue by danroth27** [dotnet/aspnetcore#33196]
Looks like Blazor WebAssembly debugging extension for VSCode doesn't have a friendly display name:

![image](https://user-images.githubusercontent.com/1874516/120379426-985cb580-c2d4-11eb-9071-ea8be77a23cd.png)

How about: "Blazor WebAssembly Debugging"?

@captainsafia

**Comment by captainsafia** [comment]
We didn't put in too much thought into the naming here since the expectation is that users should never discover this via the extensions marketplace. In fact, it should always be brought up implicitly as a dependent extension.

The long term plan was to hide it from the marketplace via some new APIs that the VS Code team is working on.

TL;DR: the idea is for the name to not matter since the goal is to make the extension "invisible" to the user.

**Comment by seanjbsol** [comment]
My 2p. If you hide the extension in VSCode, users may have trouble if something breaks. 

I like the current alert-install method as the decision whether to include this extension is borne by the user, not arbitrarily made by the extension. 

Add to that the perception some may have of Microsoft tools hiding away so they cannot be uninstalled and you could create a trust issue.

**Comment by Nick-Stanton** [comment]
Closing in favor of https://github.com/dotnet/aspnetcore/issues/46486
